/**:
  ros__parameters:
    cycle_time_ms: 10
    # Only one of the four control command topics need be specified
    # "raw", "basic"
    control_command: "basic"
    left_thruster_usb_name: "tty/USB0"
    right_thruster_usb_name: "tty/USB1"
